packages/filebrowser/src/opendialog.ts

```diff
@@ -148,7 +148,7 @@ class OpenDialog
         widget: new ToolbarButton({
           icon: newFolderIcon,
           onClick: () => {
-            browser.createNewDirectory();
+            void browser.createNewDirectory();
```

Comment:
Out of curiosity, is `OpenDialog` still used? I can't seem to figure out how load any of this in the UI.